src/packagedcode/jar_manifest.py

```diff
@@ -369,10 +339,67 @@ def dget(s):
     # Deps:
     # Require-Bundle
 
-    package['notes'] = dget('Comment')
+    comment = dget('Comment')
+
+    if comment or doc_url:
+        package["extra_data"] = {}
+        if comment:
+            package["extra_data"]['notes'] = comment
```

Comment:
we need to find a better way than using extra_data.

Reply:
See https://github.com/nexB/scancode-toolkit/pull/3447#discussion_r1262231378. We might want to add fields to the PackageData class then. And we might handle this separately, AFAIK this data is not really used anywhere and this is just a fix to make sure MANIFEST parsing returns everything else, and not crash.